project/Dependencies.scala

```diff
@@ -72,7 +72,8 @@ object Dependencies {
   val scalatestV = "3.0.2"
   val scalazV = "7.2.17"
   val scoptV = "3.6.0"
-  val shapelessV = "2.3.2"
+  val shapelessV = "2.3.3"
+  val simulacrumV = "0.12.0"
   val slf4jV = "1.7.24"
   val slickV = "3.2.3"
   val slickCatsV = "0.7-MF"
```

Comment:
You might need to update this one too. See table in the readme here https://github.com/RMSone/slick-cats

Reply:
Perhaps we should install this :) https://github.com/rtimush/sbt-updates

Reply:
Hipster here with it installed already on my system. *Please* do not add it to the project. These and other "global" level plugins can fubar a project with their sometimes weird dependencies.

> Add the following line to one of these files:
>
> ...
> Your global file at ~/.sbt/0.13/plugins/sbt-updates.sbt (for SBT 0.13.x series) or at ~/.sbt/1.0/plugins/sbt-updates.sbt (for SBT 1.x series) 

FYI I also have [sbt-dependency-graph](https://github.com/jrudolph/sbt-dependency-graph) installed locally for figuring out WTF transitive dependency messes up my builds